exercises/concept/calculator-conundrum/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+In this exercise you will be building error handling for a simple integer calculator. 
+To make matters simple, methods for calculating addition, multiplication and division are provided.
+
+The goal is to have a working calculator that returns a String with the following pattern: `16 + 51 = 67`, when provided with arguments `16`, `51` and `+`.
+
+
+```java
+CalculatorConundrum obj = new CalculatorConundrum();
+
+obj.calculate(16, 51, "+"); // => returns "16 + 51 = 67"
+
+obj.calculate(32, 6, "*"); // => returns "32 * 6 = 192"
+
+obj.calculate(512, 4, "/"); // => returns "512 / 4 = 128"
+```
+
+## 1. Implement the method calculate to support a few basic operations 
+
+The main method for implementation in this task will be the `CalculatorConundrum.calculate()` method. 
+It takes three arguments. 
+The first two arguments are integer numbers on which an operation is going to be conducted. 
```

Comment:
Slightly better wording choice: is going to operate

It seems slightly better to say that an operation is going to operate rather than the more passive voice "to be conducted"